src/ssh/cockpitsshrelay.c

```diff
@@ -2094,6 +2097,11 @@ cockpit_ssh_relay_start (CockpitSshRelay *self,
     .channel_exit_status_function = on_channel_exit_status,
   };
 
+  lang = g_getenv ("LANG");
+  command = g_strdup_printf ("LANG=%s %s",
+                             lang ? lang : "C",
+                             self->ssh_data->ssh_options->command);
```

Comment:
This should copy the other LC_* categories as well. However, I wonder if that's really neccessary - The `ssh` command already copies the entire locale environment  from its own local env to the remote end.

Reply:
We aren't using the ssh command, i don't think libssh transfers all environment variables. There is mechanism for setting them but it requires custom sshd configuration that isn't usually set by default. I'll add LC_MESSAGES here but don't think we should mess with other ones.